controllers/component_build_controller_pac.go

```diff
@@ -670,6 +673,49 @@ func checkMandatoryFieldsNotEmpty(config map[string][]byte, mandatoryFields []st
 	return nil
 }
 
+func pacRepoAddCustomParams(repository *pacv1alpha1.Repository, namespace *corev1.Namespace) bool {
+	var params []pacv1alpha1.Params
+	if repository.Spec.Params == nil || len(*repository.Spec.Params) == 0 {
+		params = make([]pacv1alpha1.Params, 0)
+	} else {
+		params = *repository.Spec.Params
+	}
+
+	// Avoid adding duplicate params potentially
```

Comment:
I'd +1 the overriding, otherwise you could end up with a Repository where the `appstudio.redhat.com/workspace_name` label value doesn't match the `appstudio_workspace` param value

Reply:
But then again, if somebody created the Repository that way maybe that's intentional :thinking:

Reply:
I think it's supposed to be set via the label, so we should stick to the setting.